components/UsMap.js

```diff
@@ -0,0 +1,130 @@
+    this.usData = usData;
+    this.height = 500; (64, 0, 64)
+    this.width = 960;
+    this.colorRange = [
+      // lightest
+      "rgb(168,0,168)",
+      "rgb(128,0,128)",
+      "rgb(98,0,98)",
+      "rgb(78,0,78)",
+      "rgb(58,0,58)"
+    ]
+    this.renderMap();
+  }
+
+  mapScoresToStates(scoreData, usData) {
+
+    for (var i = 0; i < scoreData.length; i++) {
+      let scoreStateName = scoreData[i].state;
+      var stateScores = scoreData[i].score;
+      for (var j = 0; j < usData.features.length; j++) {
+        var jsonStateName = usData.features[j].properties.name;
+        if (scoreStateName == jsonStateName) {
+          usData.features[j].properties.score = stateScores;
+          break;
+        }
+      }
+    }
+    return usData
+  }
+
+  renderMap() {
+
+    // just unpacking for tidier variable names downstream
+    let usData = this.usData;
+    let statesLived = this.statesLived;
+    let colorRange = this.colorRange;
+    let width = this.width;
+    let height = this.height;
+    let stateScores = this.stateScores;
+
+    // add the values to the state objects
+    // usData = this.mapStatesToValues(statesLived, usData);
+    usData = this.mapScoresToStates(stateScores, usData)
+
+    // initalize the pojection and path
+    let projection = d3
+      .geo
+      .albersUsa()
+      .translate([width / 2, height / 2])
+      .scale([1000]);
+
+    let path = d3
+      .geo
+      .path()
+      .projection(projection);
+
+    let color = d3
+      .scale
+      .linear()
+      .range(colorRange);
+
+    let tooltip = d3
+      .select('body')
+      .append('div')
+      .attr('id', 'tooltip')
+      .attr('style', 'position: absolute; opacity: 0;');
+
+    d3.select("body")
+      .append("svg")
+      .attr("width", width)
+      .attr("height", height)
+      .selectAll("path")
+      .data(usData.features)
+      .enter()
+      .append("path")
+      .attr("d", path)
+      .style("stroke", "#fff")
+      .style("stroke-width", "1")
+      .style("fill", function (d) {
+        return color(d.properties.score)
+      }).on('mouseover', function (d) {
+        d3.select(this)
+          .style({ opacity: '0.75' })
+          .append("text")
+          .text(function (d) {
+            console.log(d.properties.name)
+            console.log(d.properties.score)
+            return d.properties.score
+          });
+
+        tooltip
+          .transition()
+          .duration(200)
+          .style('opacity', 1)
+          .text(d.properties.name + ': ' + d.properties.score)
+
+      }).on('mouseout', function (d) {
+        d3.select(this).style({ opacity: '1.0' });
+      }).on("click", function (d) {
+        tooltip
+          .transition()
+          .duration(200)
+          .style('opacity', 1)
+          .text("Click detected on " + d.properties.name + ", create card here")
+      });
+
+  }
+
+  render() {
+    return <div id={"#" + this.props.id
+    } ></div >
```

Comment:
NIT: No need for a line break here, the whole <div> can be on one line

Reply:
Ugly right? That was my linter - it doesn't object when I change it though